source/dialogs/combi.c

```diff
@@ -253,17 +250,32 @@ static char * combi_mgrv ( const Mode *sw, unsigned int selected_line, int *stat
             char       * retv;
             char       * str  = retv = mode_get_display_value ( pd->switchers[i].mode, selected_line - pd->starts[i], state, attr_list, TRUE );
             const char *dname = mode_get_display_name ( pd->switchers[i].mode );
+			char *format_str = g_strdup( config.combi_display_format );
             if ( !config.combi_hide_mode_prefix ) {
+				if ( !(*state & MARKUP) && config.markup_combi ) {
+					// Mode does not use markup, but we want to, so escape output
+					char * tmp_str = g_markup_escape_text( str, -1 );
+					g_free(str);
+					str = tmp_str;
+					*state |= MARKUP;
+				}
+				if ( (*state & MARKUP) && !config.markup_combi ) {
+					// Mode does use markup, but we did not want to, so escape pattern string
+					char * tmp_str = g_markup_escape_text( format_str, -1 );
+					g_free(format_str);
+					format_str = tmp_str;
+				}
```

Comment:
This should take care of the escaping.

Reply:
To avoid confusion, wouldn't it be better to remove the markup_combi option and always do markup (and convert element that is not markup). 

So the combi_display_format option is always a markup string.  (don't think this is unreasonable, and how drun-display-format works)

Reply:
Done! If you think the other two options are OK, I‘ll try to put some documentation in the man page.